samples/nrf9160/mqtt_simple/overlay-tls.conf

```diff
@@ -0,0 +1,8 @@
+# To build the mqtt simple sample with TLS support
```

Comment:
Same story here, please add copyright notice.

The README.rst has already a section on how to add TLS support to the sample manually, which IMO is no longer required after this update. Perhaps we should replace it with information about overlays:
https://github.com/nrfconnect/sdk-nrf/blob/master/samples/nrf9160/mqtt_simple/README.rst#references

Reply:
Sound good. I will update the documentation to reflect the changes that have been made, and remove/rephrase how the external link regarding TLS support is added. It makes sense to link to the external TLS doc for potential issues related to TLS. If people have issues they can post comments there.